packages/effects-core/src/composition-source-manager.ts

```
@@ -77,26 +78,25 @@ export class CompositionSourceManager implements Disposable {
       ...composition,
-      id,
-      duration,
-      name,
-      endBehavior: isNaN(endBehavior) ? spec.EndBehavior.freeze : endBehavior,
-      // looping,
-      items,
-      camera,
-      startTime,
     };
+
+    if (isNaN(compositionData.endBehavior)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)